packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -874,6 +876,26 @@ order by (row_number() over (partition by 1)) desc`;
     this.lock("selectCursor");
     this.lock("select");
   }
+  buildChild() {
+    const options = { supportsJSONB: this.supportsJSONB };
+    const child = new QueryBuilder(options, this.context, this.rootValue);
+    child.parentQueryBuilder = this;
+    return child;
+  }
+  buildNamedChildFrom(name: string, from: SQLGen, field: SQLGen) {
+    if (this.children.has(name)) {
+      throw new Error(`QueryBuilder already has a child named ${name}`);
+    }
+    const child = this.buildChild();
+    child.from(from);
+    child.select(field, "_");
```

Comment:
Let's use a more readable alias here. We use `"value"` elsewhere, but if we could come up with something sensible that was easier to search for that'd be great. Failing that, value is fine:

```suggestion
    child.select(field, "value");
```